File: TrasladoDatos/ApiTestTrasladoDatos/ApiTestTrasladoDatos/Models/PaTblDocumentoEstructuraM.cs
namespace ApiTestTrasladoDatos.Models
{
    public class PaTblDocumentoEstructuraM
    {
        public int Consecutivo_Interno { get; set; }
        public string Estructura { get; set; }
        public string UserName { get; set; }
        public DateTime Fecha_Hora { get; set; }
        public int Tipo_Estructura { get; set; }
        public int Estado { get; set; }
        public string? M_UserName { get; set; }
        public DateTime? M_Fecha_Hora { get; set; }
        public Guid Id_Unc {  get; set; }


    }
}

File: MigrarSql/migrar_sql/lib/models/PaTblDocumentoEstructuraM.dart
class PaTblDocumentoEstructuraM {
  final int consecutivoInterno;
  final String estructura;
  final String userName;
  final DateTime fechaHora;
  final int tipoEstructura;
  final int estado;

  PaTblDocumentoEstructuraM({
    required this.consecutivoInterno,
    required this.estructura,
    required this.userName,
    required this.fechaHora,
    required this.tipoEstructura,
    required this.estado,
  });

  factory PaTblDocumentoEstructuraM.fromJson(Map<String, dynamic> json) {
    return PaTblDocumentoEstructuraM(
      consecutivoInterno: json['consecutivo_Interno'] as int,
      estructura: json['estructura'] as String,
      userName: json['userName'] as String,
      fechaHora: DateTime.parse(json['fecha_Hora']),
      tipoEstructura: json['tipo_Estructura'] as int,
      estado: json['estado'] as int,
    );
  }
}
